tests/FSharp.Compiler.ComponentTests/EmittedIL/SkipLocalsInit.fs

```diff
@@ -0,0 +1,133 @@
+﻿// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace FSharp.Compiler.ComponentTests.EmittedIL
+
+open Xunit
+open FSharp.Test.Utilities.Compiler
+
+#if NETCOREAPP
```

Comment:
That should be good testing wise, I'll see what we can do with the x-framework IL validation.